sw/device/lib/dif/dif_pinmux.c

```diff
@@ -0,0 +1,147 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+#include "sw/device/lib/dif/dif_pinmux.h"
+
+#include <stdbool.h>
+#include <stddef.h>
+#include <stdint.h>
+
+#include "pinmux_regs.h"  // Generated.
+#include "sw/device/lib/base/mmio.h"
+
+// TODO - autogenerate?
+#define PINMUX_REGISTER_WIDTH_IN_BYTES (PINMUX_PARAM_REG_WIDTH / 8)
+
+// TODO - autogenerate?
+#define PINMUX_INSEL_RESET_DEFAULT 0
+#define PINMUX_OUTSEL_RESET_DEFAULT 0x20820820
```

Comment:
This does not match the values in the technical specification page. Shouldn't this be `0x2082082`? I guess the last `0` is a typo.

Reply:
Thank you! Done.